server/apiv2/handlers/keyspace.go

```diff
@@ -42,6 +42,12 @@ func RegisterKeyspace(r *gin.RouterGroup) {
 	router.PATCH("/:name/config", UpdateKeyspaceConfig)
 	router.PUT("/:name/state", UpdateKeyspaceState)
 	router.GET("/id/:id", LoadKeyspaceByID)
+
+	// The following apis uses custom action and are considered deprecated.
+	router.POST("/:name/update-config", UpdateKeyspaceConfig)
+	router.POST("/:name/enable", EnableKeyspace)
+	router.POST("/:name/disable", DisableKeyspace)
+	router.POST("/:name/archive", ArchiveKeyspace)
```

Comment:
We probably don't need these

Reply:
If we are going to use master for serverless, can we remove it?